### PR TITLE
Upgraded wharf-api-client-go to v2.2.1

### PR DESCRIPTION
**Pull request by jilleJr**
- \[x] I've added a new note in the `CHANGELOG.md` file, according to docs:
  https://iver-wharf.github.io/#/development/changelogs/writing-changelogs

## Summary

- Upgraded wharf-api-client-go

## Motivation

Includes the fix for port requirement in wharfapi.Client.APIURL for gRPC communication

Issue: https://github.com/iver-wharf/wharf-api-client-go/issues/47
PR: https://github.com/iver-wharf/wharf-api-client-go/pull/48

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -226,7 +225,10 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 
 - Changed versions of numerous dependencies:
 
+  <!--lint ignore maximum-line-length-->
+
```
Comment (Alexamakans) [inline CHANGELOG.md:229]: Is this scoped to the current indent level?

**Review by Alexamakans — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (jilleJr) [reply CHANGELOG.md:229]: Yes. Or rather, the following block.

`<!--lint disable` will disable until reenabled
`<!--lint ignore` will disable for only the next block

I was a bit surprised that it counted the entire list as one block.

Would changing to `disable`+`enable` make it clearer, or OK to leave it?

Comment (jilleJr) [reply CHANGELOG.md:229]: https://github.com/remarkjs/remark-lint#ignore-warnings

Comment (Alexamakans) [reply CHANGELOG.md:229]: OK to leave imo 👍🏻

**Pull request merged**